app/src/main/java/org/p2p/wallet/svl/model/ReceiveViaLinkMapper.kt

```diff
@@ -0,0 +1,46 @@
+package org.p2p.wallet.svl.model
+
+import android.content.Context
+import org.p2p.core.common.DrawableContainer
+import org.p2p.core.common.TextContainer
+import org.p2p.core.token.Token
+import org.p2p.uikit.utils.image.ImageViewCellModel
+import org.p2p.uikit.utils.text.TextViewCellModel
+import org.p2p.wallet.R
+import org.p2p.wallet.utils.DateTimeUtils
+
+class ReceiveViaLinkMapper {
+
+    fun mapTokenAmount(token: Token.Active): TextViewCellModel =
+        TextViewCellModel.Raw(
+            text = TextContainer("${token.total} ${token.tokenSymbol}")
+        )
+
+    fun mapSenderAddress(address: String): TextViewCellModel =
+        TextViewCellModel.Raw(
+            text = TextContainer(address)
+        )
+
+    fun mapLinkCreationDate(context: Context): TextViewCellModel {
+        val dateText = DateTimeUtils.getDateFormatted(System.currentTimeMillis(), context)
```

Comment:
don't forget to put real date here

Reply:
The real date is today. So that one is correct